_metadata/repos.conf.d/centos/provider.conf

```diff
@@ -1,29 +1 @@
-2*
-3*
-4*
-5*
-6.0*
-6.1*
-6.2*
-6.3*
-6.4*
-6.5*
-6.6*
-6.7*
-6.8*
-7.0.*
-7.1.*
-7.2.*
-*atomic/
-*centosplus/
-*cloud/
-*configmanagement/
-*cr/
-*dotnet/
-*isos/
```

Comment:
Eric and I discussed grabbing ISOs and cloud-ready images directly from repos that offer that capability, and then through the use of a series of symlinks, creating images-only URLs that consolidate the images in one place.

Reply:
When it comes to centos I would not grab anything older than version 6